config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
+
+  resources :articles
```

Comment:
please add `only: [:show, :index, :new, :create]` until you actually need the other routes